From e0507a1c4936f9423fd80c427de435e91053b0eb Mon Sep 17 00:00:00 2001
From: Xulin Sun <xulin.sun@windriver.com>
Date: Tue, 30 Mar 2021 15:23:43 +0800
Subject: [PATCH] server: replace getrawmonotonic() with ktime_get_raw_ts64()

Since the commit fcae40c99fb3d09 (ALSA: Replace timespec with timespec64)
Replace the getrawmonotonic() with ktime_get_raw_ts64()

Upstream-Status: Inappropriate [upstream is still using k5.4 version]

Signed-off-by: Xulin Sun <xulin.sun@windriver.com>
---
 services/server/env/linux/osfunc.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/services/server/env/linux/osfunc.c b/services/server/env/linux/osfunc.c
index 91d66e7..59c51dc 100644
--- a/services/server/env/linux/osfunc.c
+++ b/services/server/env/linux/osfunc.c
@@ -620,9 +620,9 @@ PVRSRV_ERROR OSClockMonotonicus64(IMG_UINT64 *pui64Time)
 
 IMG_UINT64 OSClockMonotonicRawns64(void)
 {
-	struct timespec ts;
+	struct timespec64 ts;
 
-	getrawmonotonic(&ts);
+	ktime_get_raw_ts64(&ts);
 	return (IMG_UINT64) ts.tv_sec * 1000000000 + ts.tv_nsec;
 }
 
-- 
2.17.1

